# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../luatar/compile-fix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- luatar-0.0/Makefile	2005-09-14 02:26:44.000000000 +0200
+++ luatar-0.0-t2/Makefile	2006-02-01 22:22:45.000000000 +0100
@@ -52,7 +52,7 @@
 LIBS   += -ltar
 else
 LIBTAR_LIB = $(LIBTAR)/lib/libtar.a
-CFLAGS += -I$(LIBTAR)/lib
+CFLAGS += -I$(LIBTAR)/lib -I$(LIBTAR)/listhash
 LIBS   += $(LIBTAR_LIB)
 endif
 
